src/http/chain-registry/ChainRegistry.test.ts

```diff
@@ -26,6 +28,11 @@ describe('ChainRegistry', () => {
     chainRegistry = new ChainRegistry('endpoint:8081')
   })
 
+  afterEach(() => {
+    jest.clearAllMocks()
```

Comment:
Clearing the mocks is almost always a code smell that the tests are not done independently. You should in principle never need this

Reply:
I do not know much about Jest but reading around, I understand this helps to keep mocks and spies independent as it clears after a test is completed and I have seen it as a good practice because when I make a mistake in one call `HasBeenCalled...` it displays each of the options for every test unit instead of the one from my current scope. But we can discuss it offline